config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -107,13 +116,19 @@ periodics:
 
               apt-get update && apt-get install -y ansible
 
+              # Details of IBM S3 storage
+              S3_SERVER=s3.us-south.cloud-object-storage.appdomain.cloud
+              BUCKET=ppc64le-ci-builds
+              DIRECTORY=ci
+
+              # Build and push artifacts to IBM COS
+              REPOROOT=/home/prow/go/src/github.com/kubernetes/kubernetes
+              kubetest2 tf --build --repo-root $REPOROOT --stage cos://us-south/$BUCKET/$DIRECTORY
```

Comment:
part of the `--build` I also copy the kubectl and other binaries to the _rundir, hope running these build and up commands separately uses same copied kubectl from the _rundir directory.

Reply:
Yes the test binaries are being referred from same RunDir for `--build` and `--test` calls.
I see below trace in job log:
```
+ kubetest2 tf --build --repo-root /home/prow/go/src/github.com/kubernetes/kubernetes --stage cos://us-south/ppc64le-ci-builds/ci
I1105 05:16:41.853769   11602 app.go:61] The files in RunDir shall not be part of Artifacts
I1105 05:16:41.853798   11602 app.go:62] pass rundir-in-artifacts flag True for RunDir to be part of Artifacts
I1105 05:16:41.853814   11602 app.go:64] RunDir for this run: "/home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd"
----------------
-------------
--------------
+ kubetest2 tf --powervs-dns k8s-tests --powervs-image-name centos-9-stream-20Gb --powervs-region syd --powervs-zone syd05 --powervs-service-id c1c8f2a5-a35a-4e32-a1ba-4c1f8238742f --powervs-ssh-key powercloud-bot-key --ssh-private-key /etc/secret-volume/ssh-privatekey --s3-server s3.us-south.cloud-object-storage.appdomain.cloud --bucket ppc64le-ci-builds --directory ci --extra-vars=k8s_tar_bundles:kubernetes-server-linux-ppc64le.tar.gz --build-version v1.32.0-alpha.3.247+e5a31e8bbc6785 --release-marker v1.32.0-alpha.3.247+e5a31e8bbc6785 --cluster-name config1-1730786361 --workers-count 2 --up --auto-approve --retry-on-tf-failure 3 --break-kubetest-on-upfail true --powervs-memory 32 --test=ginkgo -- --parallel 10 --use-built-binaries true '--focus-regex=\[Conformance\]' '--skip-regex=\[Serial\]'
I1105 05:59:22.878817   80908 app.go:61] The files in RunDir shall not be part of Artifacts
I1105 05:59:22.878850   80908 app.go:62] pass rundir-in-artifacts flag True for RunDir to be part of Artifacts
I1105 05:59:22.878868   80908 app.go:64] RunDir for this run: "/home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd"
I1105 05:59:22.884004   80908 app.go:136] ID for this run: "ea49f7ea-ca62-46e9-85e5-1aea196a82fd"
------------------
---------------
-------------
KUBECONFIG set to: /home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/config1-1730786361/kubeconfig
I1105 06:14:12.927854   80908 deployer.go:331] About to run: [kubectl get nodes -o=name]
I1105 06:14:14.013218   80908 deployer.go:268] Dumping cluster info..
I1105 06:14:14.013312   80908 dumplogs.go:12] Collecting cluster logs under /home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd/logs
I1105 06:14:14.013542   80908 dumplogs.go:36] About to run: [kubectl cluster-info dump]
I1105 06:14:22.083116   80908 dumplogs.go:44] Executed [kubectl cluster-info dump] successfully
I1105 06:14:22.151309   82358 ginkgo.go:131] Using kubeconfig at /home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/config1-1730786361/kubeconfig
I1105 06:14:22.151374   82358 ginkgo.go:100] Running ginkgo test as /home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd/ginkgo [--nodes=10 /home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd/e2e.test -- --kubeconfig=/home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/config1-1730786361/kubeconfig --kubectl-path=/home/prow/go/src/github.com/ppc64le-cloud/kubetest2-plugins/_rundir/ea49f7ea-ca62-46e9-85e5-1aea196a82fd/kubectl --ginkgo.skip=\[Serial\] --ginkgo.focus=\[Conformance\] --report-dir=/logs/artifacts --ginkgo.timeout=24h0m0s]
```

Reply:
okay, thanks